sdk/src/main/java/com/uid2/network/DataEnvelope.kt

```diff
@@ -12,21 +13,19 @@ import javax.crypto.spec.SecretKeySpec
  * **See Also:**
  * [GitHub](https://github.com/IABTechLab/uid2docs/blob/main/api/v2/getting-started/gs-encryption-decryption.md)
  */
-public object DataEnvelope {
-    // The name and transformation of the encryption algorithm used.
-    private const val ALGORITHM_NAME = "AES"
-    private const val ALGORITHM_TRANSFORMATION = "AES/GCM/NoPadding"
+public class DataEnvelope {
```

Comment:
Why did this become a class instead of an object? by looking at it, it seems it holds no state

Reply:
`DataEnvelope` is mocked in `UID2ClientTest` so that I can return fake data when asked to decrypt.

Reply:
Addressed with commits shared by Nacho!